sql/handler.h

```diff
@@ -963,11 +971,12 @@ struct xid_recovery_member
   bool decided_to_commit;
   Binlog_offset binlog_coord; // semisync recovery binlog offset
   XID *full_xid;           // needed by wsrep or past it recovery
+  ulong server_id;         // server id of orginal server
```

Comment:
While I was assuming (before I checked indeed) the new member is of the same type as the global var of `server_id`,
the declaration like
`decltype(::server_id) server_id;`
would speak it the most explicit way.

Reply:
Replaced all ulong to decltype in commit.